### ISSUE TITLE
Empty lines unexpectedly added after table tags

### ISSUE THREAD

**Issue by thorn0** [prettier/prettier#5810]
**Prettier 1.16.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeGBDARgGzgPgB0oACE9AJ31QHoYrjbNcDGATASwDdqbOeQAGhAQADjA7QAzslAYKFCAHcACvIQyUGLhA5shILBQxgA1nBgBlUSY5QA5snoBXOMIAWMALY4A6u454KRswOEsNQO5AgE9kcCkZYTspOAoYFWN7LwxkADMMHBThACspAA8AIWMzC0sMLzgAGTs4PIKikFKyyzt7PABFZwh4NsK3EBsKFIo4zx8DUQo7GF89GHdkAA4ABmFFiBTfY1E4xbhprlbhCjgAR2cOG4yMLJykfLHhFK8OJwpXL69AZDEbvdrjZirNjrZAAJmE9AwHBwvQAwhAvNk4lBoFcQM4UgAVbCaD4pAC+5KAA)
```sh
--parser html
```

**Input:**
```html
<table>
  <tr></tr>
</table>
<div></div>
```

**Output:**
```html
<table>
  <tr></tr>
</table>

<div></div>

```

**Expected behavior:**
```html
<table>
  <tr></tr>
</table>
<div></div>

```

Happens only if `tbody` tags are omitted.

**Comment by ikatyang** [comment]
The issue here is that the position of `</table>` was eaten by the pseudo node `<tbody>`.

**Comment by ikatyang** [comment]
Pseudo `<tbody>` for `<tr>` is removed in [angular-html-parser@1.4.0](https://github.com/ikatyang/angular-html-parser/blob/master/packages/angular-html-parser/CHANGELOG.md#140-2020-01-28), updating the parser should fix the issue.

**Comment by thorn0** [comment]
closed by #7461

**Comment by thorn0** [comment]
Seems like some cleanup is needed here, see https://codecov.io/gh/prettier/prettier/compare/07b9465dcfeb7228eef9dff5a959dea645854697...f40fcba710f7a0d4e82e5458334dca97d994c94a/src/src/language-html/parser-html.js

**Comment by thorn0** [comment]
closed by #7468
